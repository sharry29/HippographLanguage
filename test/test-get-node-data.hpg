int main() {
  node<string:string> n1 = "abc":"def";
  node<string:int> n2 = "abc":42;
  node<int:string> n3 = 42:"abc";
  node<int:int> n4 = 42:1;

  string d1 = n1.get_data();
  print(d1);

  int d2 = n2.get_data();
  print_int(d2);

  string d3 = n3.get_data();
  print(d3);

  int d4 = n4.get_data();
  print_int(d4);

  return 0;
}