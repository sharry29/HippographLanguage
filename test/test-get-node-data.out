def
42
abc
1